src/lib.rs

```diff
@@ -183,7 +182,8 @@ impl Decimal {
 
         let old_scale = self.scale();
 
-        if dp < old_scale && dp < 20 { // Technically, it's 28...
+        if dp < old_scale && dp < 20 {
+            // Technically, it's 28...
```

Comment:
The comment doesn't have much context in the first place.  I don't think the change from rustfmt made things any better or worse.